Frontend/src/components/usermanagement/SettingsDeleteUser.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div>
+      <p>{{$t('settings.deleteQuestion')}} </p>
+     <v-dialog
+      v-model="dialog"
+      width="500"
+    >
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn
+          color="red lighten-2"
+          dark
+          v-bind="attrs"
+          v-on="on"
+        >
+         {{$t('settings.deleteAccount')}}
+        </v-btn>
+      </template>
+
+      <v-card>
+        <v-card-title class="headline grey lighten-2">
+         {{$t('settings.deleteSure')}}
+        </v-card-title>
+
+        <v-card-text>
+          {{$t('settings.deleteHint')}}
+        </v-card-text>
+
+        <v-divider></v-divider>
+
+        <v-card-actions>
+          <v-spacer></v-spacer>
+          <v-btn
+            color="primary"
+            text
+            @click="deleteUserComplete"
+          >
+            {{$t('settings.delete')}}
+          </v-btn>
+          <v-btn
+            color="primary"
+            text
+            @click="dialog = false"
+          >
+            {{$t('settings.dontDelete')}}
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+    <p v-if="deletedUser">{{$t('settings.deletedUser')}}</p>
```

Comment:
Das sollte doch in in einem Toast angezeigt werden, sonst sieht man außer der Weiterleitung zur LandingPage kein Feedback, oder?